src/index.css

```
@@ -1,37 +1,44 @@
-  --color-surface-sidebar: #070D1A;
-
-  /* Borders — blue-tinted, not white */
-  --color-border-subtle: rgba(56, 78, 128, 0.25);
-  --color-border-default: rgba(56, 78, 128, 0.35);
-
-  /* Brand colors */
-  --color-brand-primary: #C6B6D4;
-  --color-brand-secondary: #4F46E5;
-  --color-brand-cyan: #22D3EE;
-  --color-brand-teal: #14B8A6;
+  --font-sans: 'Geist', ui-sans-serif, system-ui, -apple-system, 'Segoe UI', Helvetica, Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)